Gemfile

```diff
@@ -53,3 +53,5 @@ group :test do
   gem 'timecop'
 end
 
+
+gem "rinda", "~> 0.2.0"
```

Comment:
We shouldn't need this change as the gemspec is already loaded by the gemfile 

https://github.com/rapid7/metasploit-framework/blob/b29fbe45a2cb0e0683ace4f15635538acc2eac39/Gemfile#L4